mitmproxy/net/http/cookies.py

```diff
@@ -114,11 +114,11 @@ def _read_cookie_pairs(s, off=0):
         lhs, off = _read_key(s, off)
         lhs = lhs.lstrip()
 
-        if lhs:
-            rhs = None
-            if off < len(s) and s[off] == "=":
-                rhs, off = _read_value(s, off + 1, ";")
+        rhs = None
+        if off < len(s) and s[off] == "=":
```

Comment:
As @kajojify mentioned in #2847 , initializing `rhs` as `None` will throw an error when the cookie is `"one"` which as you can see from the test cases will parsed as `["one", None]`

Reply:
Thanks for pointing that out. I think we should initialize it as an empty string then.

Reply:
Hi, I was already working on it, I am so sorry about this.

Reply:
If you initialise `rhs` as an empty string, then you need to find a way to not append in the case of an empty cookie , but if there is a cookie like this `=` you need to parse it in a way that it looks like `[["", ""]]` . I am not working on the issue you are trying to solve, sorry for the confusion